15d507c4837efc7462d22460d4966f243f4daed61fe210775ae268bd74454d50.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
@phillewis Please move this file into the registry subfolder.

Reply:
@AndrewWestberg Hahaha, sorry about that.  I've been driving tired for the past few days.
New commit with the file moved to the registry folder.